src/server/plugins/router.ts

```diff
@@ -32,9 +44,18 @@ export default {
         },
         {
           method: 'get',
-          path: '/help/privacy',
-          handler(_request: Request, h: ResponseToolkit) {
-            return h.view('help/privacy')
+          path: '/help/privacy/{slug?}',
+          handler(request: Request, h: ResponseToolkit) {
+            const { slug } = request.params
+
+            const privacyPolicy = privacyPolicies[slug as string]
+
+            if (!privacyPolicy) {
```

Comment:
The compiler was kicking off a bit because `slug` might be `undefined`

So if we do both checks it'll be happy

```suggestion
            if (!slug || !(slug in privacyPolicies)) {
```